### DISCUSSION TITLE
How to add data provider without UI?

### DISCUSSION THREAD

**Discussion by v3ss0n** [Budibase/budibase#7429] [General]
I just want a List of name from users - which would fill into the combo boxs' data-source
Can we do that without adding a data provider as parent ? I want to do without interefecing with UI.

**Comment by andz-bb** [comment]
Hey @v3ss0n - when talking about the UI do you mean the page controls at the bottom of the data provider? If you just need to populate an Options Picker within a Data Provider, you could disable pagination in the Data Provider options. This is one I've made and the only UI rendered is the picker component itself.

![picker in data provider](https://user-images.githubusercontent.com/110921612/186417871-4821b2b5-83ea-437f-b89d-e03888cd8d0d.png)

**Reply by v3ss0n** [reply]
Thanks that works , but somethines it takes a space even without pager.
